src/odb/include/odb/dbStream.h

```diff
@@ -178,21 +159,17 @@ class dbOStream
     } else {
       int l = strlen(c) + 1;
       *this << l;
-      int n = fwrite(c, l, 1, _f);
-      if (n != 1) {
-        write_error();
-      }
+      _f.write(c, l);
     }
 
     return *this;
   }
 
   dbOStream& operator<<(dbObjectType c)
   {
-    int n = fwrite(&c, sizeof(c), 1, _f);
-    if (n != 1) {
-      write_error();
-    }
+    char dst[sizeof(c)];
+    std::memcpy(dst, &c, sizeof(c));
+    _f.write(dst, sizeof(c));
```

Comment:
This is using exceptions, which is good, but exceptions for this file handle needs to be enabled below.

I'm a bit mystified by the std::memcpy(). Perhaps break out a function with generics(to take any type) and document in that function why std::memcpy() is used?